docs/source/experiment/WebUI.rst

```diff
@@ -0,0 +1,375 @@
+WebUI
+#####
+
+
+Q&A
+***
+
+There are many trials in the detail table but ``Default Metric`` graph is empty.
+================================================================================
+
+First you should know that ``Default metric`` and ``Hyper parameter`` graph only show succeeded trials.
+
+
+What should you do when you think the graph is strange, such as ``Default metric``, ``Hyper parameter``...
+==========================================================================================================
+
+* Download the experiment results(``experiment config``, ``trial message`` and ``intermeidate metrics``) from ``Experiment summary`` and then upload these results in your issue.
+
+
+.. image:: ../../img/webui-img/summary.png
+   :target: ../../img/webui-img/summary.png
+   :alt: summary
+
+
+What should you do when your experiment is error
+================================================
+
+
+* Click the icon in the right of ``experiment status`` and screenshot the error message. 
+* And then click the ``learn about`` to download ``nni-manager`` and ``dispatcher`` logfile.
+* Please file an issue from the `Feedback` in the `About` and upload above message.
+
+
+.. image:: ../../img/webui-img/experimentError.png
+   :target: ../../img/webui-img/experimentError.png
+   :alt: experimentError
+
+
+What should you do when your trial is failed
+============================================
+
+* ``Customized trial`` could be used in here. Just submit the same parameters to the experiment to rerun the trial.
+
+.. image:: ../../img/webui-img/detail/customizedTrialButton.png
+   :target: ../../img/webui-img/detail/customizedTrialButton.png
+   :alt: customizedTrialButton
+
+
+
+.. image:: ../../img/webui-img/detail/customizedTrial.png
+   :target: ../../img/webui-img/detail/customizedTrial.png
+   :alt: customizedTrial
+
+
+
+* ``Log model`` will help you find the error reason. There are three buttons ``View trial log``, ``View trial error`` and ``View trial stdout`` on local mode. If you run on the OpenPAI or Kubeflow platform, you could see trial stdout and nfs log.
+  If you have any question you could tell us in the issue.
+
+  **local mode:**
+
+.. image:: ../../img/webui-img/detail/log-local.png
```

Comment:
This should have the same indent as ``local mode:``.

Reply:
fixed